src/wp-admin/includes/block-editor.php

```diff
@@ -0,0 +1,241 @@
+<?php
+/**
+ * Block Editor API.
+ *
+ * @package WordPress
+ * @subpackage Editor
+ * @since 5.8.0
+ */
+
+/**
+ * Gets the default block editor settings.
+ *
+ * @since 5.8.0
+ *
+ * @return array The default block editor settings.
+ */
+function get_default_block_editor_settings() {
+	/**
+	 * @global WP_Post $post Global post object.
+	 */
+	global $post;
```

Comment:
This could be empty in some situations like (edit-widgets, edit-site...) but maybe it's fine :)

Reply:
Yes, it's empty for some screens and I'm not quite sure what we can do about it, many filters depend on the `$post` though 😄